docs/USAGE.rst

```diff
@@ -0,0 +1,366 @@
+   
+To see an item view of an image, click on the image/item's row in the folder view. You will then go to the item view, which looks like the below screenshot, of an item named ``0579XY112001_01_13.svs`` that is located in the ``AvailableToProcess`` folder. In the info panel you can see some metadata such as the image size and WSI DeID creation date. The item view will present you with subsections for a panning/zooming ``Image Viewer``, a listing of ``Large Image Metadata``, the set of ``Associated Images``, and image/item specific ``WSI DeID Workflow`` actions.
+
+.. image:: screenshots/example_image_item_view.png
+   :height: 100
+   :width: 200
+   :alt: image item view
+
+Below is a screenshot of the action buttons available in the ``WSI DeID Workflow`` section of the ``AvailableToProcess`` folder. Different folders will present different combinations of buttons, see the Workflow Overview section above for details.
+  
+ .. image:: screenshots/wsideid_workflow_buttons_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: WSI DeID workflow buttons
+
+Importing Data
+==============
+
+The import process assumes that the system has been configured with a mounted import directory, that is, the local filesystem folder that was mounted as the import path in the docker-compose configuration.
+
+Imported Files and Folders
+--------------------------
+
+Files are automatically copied from the local import directory to the ``AvailableToProcess`` folder in the ``WSI DeID`` collection in the WSI DeID. Files can have any folder structure; the folder structure is not significant in the import process. Excel files (identified by ending in .xls or .xlsx) and image files (anything else except for ignored files) will be imported. To facilitate bulk uploads, we ignore files ending in .txt, .xml, .zip from the import process -- this list can be easily changed.
+
+Import Process
+--------------
+
+From the ``AvailableToProcess`` folder (or any sub folder) in the WSI DeID, click on the ``Import`` button, as shown in the below screenshot.
+
+.. image:: screenshots/import_button_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: import button
+
+A background process starts that scans through the mounted import directory, and does the following:
+
+- Each Excel file is parsed for a header row that has TokenID, ImageID, and ScannedFileName.
+- If there are any Excel files that do not have a header row, an error is generated and appears on the screen, and files are not imported.
+- If the same ScannedFileName is listed in multiple Excel files, the newest file is used by preference.
+- The ScannedFileName is expected to be just the file name (e.g., no folder path).
+
+After the image names and information in the metadata file are reconciled, the WSI DeID will classify images as one of the following:
+
+- ``present``: The image is listed in an Excel file and is already in the WSI DeID based on file path and matching file size. No action is performed.
```

Comment:
The use no longer sees these words.  Rather, they see one of  'Imported', 'Already imported', 'Updated' (this was 'replaced'), 'File missing', 'Not in DeID Upload file', 'Error in DeID Upload file', 'Failed to import', 'Duplicate ImageID'.

Error in DeID Upload file means the row in excel failed to validate; the reason is shown in the report.

Duplicate ImageID means that the same image ID was repeated in the excel file(s) for different filenames.